### DISCUSSION TITLE
How to create Imagemonikers with PNG and XAML

### DISCUSSION THREAD

**Discussion by RobertvanderHulst** [VsixCommunity/Community.VisualStudio.Toolkit#509] [Q&A]
I have some images in PNG format.
What would be the recommended way to convert these to Image monikers and get both XAML and PNG images?

**Comment by madskristensen** [comment]
I've never converted an image to XAML, but if you do and add them to a .imagemanifest file, then just make sure they have the same file name to make the image manifest tool see them as a single image

**Reply by madskristensen** [reply]
If you have a XAML image, I don't think it makes sense to use the PNG for anything. XAML is vector and will be higher quality than any PNG regardless of resolution and DPI

**Comment by RobertvanderHulst** [comment]
I do not have an XAML image. How would you produce that? Is there a tool that can convert a vector in a PDF to XAML?

**Reply by workgroupengineering** [reply]
Using Inkscape

**Reply by madskristensen** [reply]
PNGs are fine. That's what I use when I work with custom images. If you use .imagemanifest, you can specify the .png in multiple sizes to make it look great in all DPIs that users might use
